compiler/luci/service/src/Nodes/CircleReshape.cpp

```diff
@@ -34,4 +62,107 @@ luci::CircleNode *CloneNodeLet<CN::OPQR>::visit(const luci::CircleReshape *node)
   return cloned;
 }
 
+namespace sinf
+{
+
+/**
+ * @note  CircleReshape has new shape info in two places: 2nd input and attribute.
+ *        This shape inference uses shape from input 'shape' node when it's constant.
+ *        If not, shape will be from node itself. shape from attribute is not used.
+ *
+ * TODO Change this policy when not appropriate
+ */
```

Comment:
let's remove this comment as this PR is just migration.

Reply:
> let's remove this comment as this PR is just migration.

I've removed this comment :)